recipe/meta.yaml

```
@@ -74,13 +76,11 @@ requirements:
     # Keep both versions of openssl in sync
     - openssl
   host:
-    - {{ mpi }}  # [mpi != 'nompi']
+    - {{ mpi | replace("impi", "impi-devel") }}  # [mpi != 'nompi']
```

string parsing is "clever" and assumes that `impi` doesn't appear as a substring in other options.

```suggestion
    - {{ mpi }}   # [mpi != 'nompi' or mpi != 'impi']
    - impi-devel  # [mpi == 'impi']
```